### DISCUSSION TITLE
Show outlines/borders in media crop library

### DISCUSSION THREAD

**Discussion by alice-gothberg** [umbraco/Umbraco-CMS#11872] [Features and ideas]
I am working on a page with product images that have a white background. I noticed that, when setting media crops on these pictures, I had to set it blindly because the edges of the crop were invisible because of the white background.

The crops work perfectly for normal photos with backgrounds:
![Crop preview image 1](https://user-images.githubusercontent.com/77152859/149735037-e3583e3b-79e3-49a3-bd89-6f94fa8e3fda.png)

But for pictures with white backgrounds, the edges are invisible:
![Crop preview image white background](https://user-images.githubusercontent.com/77152859/149735108-b772297b-7ca5-419b-8aaa-642b6acc700b.png)

Is this something that could be fixed?

**Comment by markadrake** [comment]
Just a quick question - I notice that the transparent texture behind the image is not showing. Is all of the "white" in your 2nd screenshot part of the original image already? I would expect the transparent background to be showing.

**Reply by alice-gothberg** [reply]
Yeah, the image is all white so there's no transparent texture at all, it has quite a lot of space between the table and the edges and is in the same format as the crop area! The first one is in portrait mode, so I'm guessing that's why they look different.
